src/Symfony/Component/Validator/Constraints/Expression.php

```diff
@@ -45,6 +46,7 @@ class Expression extends Constraint
      * @param array<string,mixed>                              $options
      * @param bool|null                                        $negate     Whether to fail if the expression evaluates to true (defaults to false)
      */
+    #[HasNamedArguments]
```

Comment:
The options arguments also need to be deprecated, or i'm missing something ?

Reply:
good catch, should be good now